modelutils.py

```diff
@@ -91,3 +104,57 @@ def get_sequential_groups(model):
         ]
     else:
         raise ValueError(MODEL_ERROR_MSG.format(model.config.model_type))
+
+
+def read_quant_weight_from_file(load_path, block_i, layer_name):
+    return torch.load(load_path + '/' + str(block_i) + '/' + layer_name)
+
+
+def load_quantized_model(model, load_path):
+    layers = get_layers(model)
+    for i in range(len(layers)):
+        layer = layers[i]
+        sub_layers = find_sublayers(layer)
+        for name in sub_layers:
+            dic = read_quant_weight_from_file(load_path, i, name)
+            sub_layers[name].weight = nn.Parameter(layer_weight_dequantization(dic).to(
+                sub_layers[name].weight.data.dtype))
+        layers[i] = layer
+    model.load_state_dict(torch.load(load_path  + "/not_quantized_weights.pt"), strict=False)
+    return model
+
+
+def layer_weight_dequantization(dic):
```

Comment:
dic is not a very informative name...consider renaming. Like `config_dict`

Reply:
add a docstringx

Reply:
Thank you would be added.